Upgrade Sharp to 0.30.3 (latest version)

### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

Related to https://github.com/directus/directus/issues/4424#issuecomment-792730665

Sharp has released an update which addresses this issue.

### To Reproduce

While on an M1 Mac:

1. `docker run --rm -it node:16 /bin/bash`
2. `npm install --global directus`
3. Installation fails, see errors shown section. Even installing libvips-dev won't fix it.

Compare this to installing latest sharp (0.30.3 at the moment):

1. `docker run --rm -it node:16 /bin/bash`
2. `npm install --global sharp@latest`
3. Installation succeeds.

---

Sidenote: I'd provide a PR myself but ran into another M1 related issue due to node-oracledb that I did not want to deal with:

```
npm ERR! oracledb ERR! NJS-067: a pre-built node-oracledb binary was not found for darwin arm64
npm ERR! oracledb ERR! Try compiling node-oracledb source code using https://oracle.github.io/node-oracledb/INSTALL.html#github
```

### Errors Shown

```
npm ERR! sharp: Please see https://sharp.pixelplumbing.com/install for required dependencies
npm ERR! sharp: Installation error: Use with glibc 2.28 requires manual installation of libvips >= 8.11.3
```

### What version of Directus are you using?

9.7.1

### What version of Node.js are you using?

16.14.0

### What database are you using?

N/A

### What browser are you using?

N/A

### What operating system are you using?

macOS

### How are you deploying Directus?

Docker

--- a/api/package.json
+++ b/api/package.json
@@ -150,7 +150,7 @@
 		"resolve-cwd": "^3.0.0",
 		"rollup": "^2.67.3",
 		"sanitize-html": "^2.6.0",
-		"sharp": "^0.29.0",
+		"sharp": "^0.30.3",
 		"stream-json": "^1.7.1",
 		"supertest": "^6.1.6",
 		"tmp-promise": "^3.0.3",

